manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WorldOfWarships.ModStation

```

Comment:
```suggestion
PackageIdentifier: Wargaming.WorldOfWarshipsModStation
```

Reply:
Make sure to change `manifests/w/WorldOfWarships/ModStation/2.1.0.2/WorldOfWarships.ModStation.locale.en-US.yaml` to `manifests/w/Wargaming/WorldOfWarshipsModStation/2.1.0.2/Wargaming.WorldOfWarshipsModStation.yaml` for each yaml file